### PR TITLE
Add new AIX72 machines

### PR DESCRIPTION
**Pull request by AshCripps**
Add three new AIX72 will now be used for building node in CI.

@nodejs/build-infra can I get `140.211.9.131` and `140.211.9.101` adding to the test CI firewall and `140.211.9.77` added to the release CI

### CODE REVIEW

**Comment by richardlau** [comment]
> 
> @nodejs/build-infra can I get `140.211.9.131` and `140.211.9.101` adding to the test CI firewall and `140.211.9.77` added to the release CI

Done. 

`140.211.9.101` was already in the firewall rules:
```
-A jnlp -s 140.211.9.101/32 -m comment --comment test-osuosl-aix61-ppc64_be-1 -j ACCEPT
```
I've taken out the old entry. There does seem to be quite a few old entries (e.g. anything mentioning "aix61") in the firewall rules that should be cleaned up, but I've left them alone for the moment as we'd need to verify the IP addresses weren't repurposed before removing them.

File: ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml [hunk 1]
```diff
@@ -16,15 +16,18 @@
   when: pip_exists.stat.exists == False
 
 - name: install pip
-  raw: /usr/bin/python /home/iojs/get-pip.py
+  raw: /opt/bin/python3 /home/iojs/get-pip.py
```
Comment (richardlau) [inline ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:19]: Will this work on the AIX 7.1 boxes?

```
$ ssh test-ibm-aix71-ppc64_be-4 "ls -al /opt/bin"
/opt/bin not found
$
```

Comment (AshCripps) [reply ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:19]: no but we shouldnt need to reinstall pip on those boxes so the step will be skipped

Comment (mhdawson) [reply ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:19]: hmm, we may have the AIX 6.1 boxes for a while, would be good if the ansible scripts could be used to replace one if that becomes necessary.

Comment (AshCripps) [reply ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:23]: ive added some logic for aix71 and aix72

Comment (AshCripps) [reply ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:23]: ive added some logic for aix71 and aix72

**Comment by AshCripps** [comment]
https://ci.nodejs.org/job/node-test-commit-aix-AshCripps/10/ 

Full green build this is now ready for review

File: jenkins/scripts/VersionSelectorScript.groovy [hunk 2]
```diff
@@ -92,6 +92,7 @@ def buildExclusions = [
 
   // AIX PPC64 ---------------------------------------------
   [ /aix71/,                          anyType,     lt(10)  ],
+  [ /aix72/,                          anyType,     lt(14)  ],
```
Comment (richardlau) [inline jenkins/scripts/VersionSelectorScript.groovy:95]: We probably want to ensure we only build releases on either AIX 7.1 or 7.2 but not both?

File: ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml [hunk 3]
```diff
@@ -15,16 +15,23 @@
...
    dest: "/usr/bin/pip"
    state: link
   when: pip_exists.stat.exists == False
 
 - name: install tap2junit
-  pip: name=tap2junit state=present
+  pip: 
+    name: tap2junit
+    state: present
+    executable: /opt/bin/pip
```
Comment (richardlau) [inline ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:37]: Is this supposed to be `/usr/bin/pip`?

Comment (AshCripps) [reply ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:37]: nope supposed to be `opt/freeware/bin/pip` according to the commit history

Comment (AshCripps) [reply ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml:37]: wait scratch that reading the wrong part of the code

**Review by richardlau — APPROVED** [review]

**Pull request merged**